js/everything.js

```diff
@@ -169,10 +169,32 @@ function cleanTime(time) {
     return ret;
 }
 
+function storeTime(time) {
+    var videoID = getCurrentVideoID();
+    if (window.sessionStorage && videoID) {
+        window.sessionStorage[videoID] = time;
+    }
+}
+
 function updatePlayerTime() {
-    $("#currentTime").text(cleanTime(player.getCurrentTime()));
+    var currentTime = player.getCurrentTime();
+    $("#currentTime").text(cleanTime(currentTime));
     // TODO: after the video loads, player.getDuration() may have changed +/- 1
     $("#totalTime").text(cleanTime(player.getDuration()));
+    storeTime(currentTime);
+}
+
+function loadTime() {
+    var videoID = getCurrentVideoID();
+    if (window.sessionStorage && window.sessionStorage.hasOwnProperty(videoID)) {
+        time = window.sessionStorage[videoID];
+        if (!isNaN(time)) {
+            return parseInt(time);
```

Comment:
We should change this to `parseInt(time, 10);` for some values, the base will be ambiguous and strange things can happen